apps/tasks/lib/components/patient_bottom_sheet.dart

```diff
@@ -0,0 +1,175 @@
+import 'package:flutter/material.dart';
+import 'package:helpwave_localization/localization.dart';
+import 'package:helpwave_theme/constants.dart';
+import 'package:helpwave_theme/theme.dart';
+import 'package:provider/provider.dart';
+import 'package:tasks/components/task_expansion_tile.dart';
+import 'package:tasks/dataclasses/task.dart';
+
+class PatientBottomSheet extends StatefulWidget {
+  const PatientBottomSheet({Key? key, required this.task}) : super(key: key);
+
+  final TaskWithPatient task;
```

Comment:
The Tasks for the patient will be loaded by the patient id, only require the patient